packages/vaadin-combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -221,6 +221,13 @@ class ComboBoxDropdownElement extends DisableUpgradeMixin(mixinBehaviors(IronRes
     return this.alignedAbove ? -overlayRect.height : targetRect.height;
   }
 
+  _shouldAlignLeft(targetRect) {
+    const spaceRight =
+      (window.innerWidth - targetRect.right - Math.min(document.body.scrollLeft, 0)) / window.innerWidth;
+
+    return spaceRight < 0.3;
```

Comment:
I'm not sure if I clearly understand where the `0.3` value comes from. 

I would probably suggest adding a constant with a specific name to provide some context to this value.

Reply:
Extracted.